.github/workflows/_deploy_kafka_test.yml

```diff
@@ -0,0 +1,198 @@
+          SIDECAR_IMAGE: ${{ format('{0}/{1}', inputs.sidecar-registry, inputs.key-release-image) }}
+        id: run-workload
+        run: | 
+          az aks get-credentials --name $CLUSTER_NAME --resource-group $RESOURCE_GROUP  --overwrite-existing
+          cd confidential-container-demos/kafka
+
+          SkrClientAKVEndpoint=${SkrClientAKVEndpoint#"https://"}
+          SkrClientAKVEndpoint=${SkrClientAKVEndpoint#"http://"}
+          export SkrClientAKVEndpoint=${SkrClientAKVEndpoint%%/*}
+
+          echo "SkrClientKID value is: $SkrClientKID"
+          export SkrClientKID=$SkrClientKID
+          echo "SkrClientMAAEndpoint value is: $SkrClientMAAEndpoint"
+          export SkrClientMAAEndpoint=$SkrClientMAAEndpoint
+          echo "SkrClientAKVEndpoint value is: $SkrClientAKVEndpoint"
+          export SkrClientAKVEndpoint=$SkrClientAKVEndpoint
+          echo "TOPIC value is: $TOPIC"
+          export TOPIC=$TOPIC
+
+          rm consumer-example.yaml 2>&1 || true 
+          envsubst -i consumer/consumer.yaml -o consumer-example.yaml
+          echo "Generating Security Policy for consumer"
+          cat consumer-example.yaml
+          export WORKLOAD_MEASUREMENT=$(az confcom katapolicygen -y consumer-example.yaml --print-policy | base64 --decode | sha256sum | cut -d' ' -f1)
+          # chmod +x setup-key.sh
+          # rm $SkrClientKID-info.json 2>&1 || true 
+          # rm $SkrClientKID-pub.pem 2>&1 || true 
+          # rm $SkrClientKID-release-policy.json 2>&1 || true 
+          # az account set --subscription "Azure Confidential Computing DEV AND TEST"
+          # bash setup-key.sh $SkrClientKID $SkrClientAKVEndpoint
+      
+          # sleep 2
+          # export PUBKEY=$(cat $SkrClientKID-pub.pem)
+          # rm producer-example.yaml 2>&1 || true
+          # envsubst <producer/producer.yaml> producer-example.yaml
+
+          # sed -i '25s/^/            /' producer-example.yaml
+          # sed -i '26s/^/            /' producer-example.yaml
+          # sed -i '27s/^/            /' producer-example.yaml
+          # sed -i '28s/^/            /' producer-example.yaml
+          # sed -i '29s/^/            /' producer-example.yaml
+          # sed -i '30s/^/            /' producer-example.yaml
+          # sed -i '31s/^/            /' producer-example.yaml
+          # sed -i '32s/^/            /' producer-example.yaml
+          # sed -i '33s/^/            /' producer-example.yaml
+          # sed -i '34s/^/            /' producer-example.yaml
+          kubectl delete -f consumer-example.yaml 2>&1 || true 
+          sleep 5 
+          kubectl apply -f consumer-example.yaml 2>&1 || true 
+          sleep 10
+
+          # kubectl delete -f producer-example.yaml 2>&1 || true 
+          # sleep 5 
+          # kubectl apply -f producer-example.yaml 2>&1 || true 
+
+          sleep 30 
+          export ConsumerIP=$(kubectl get svc consumer -n kafka --output=jsonpath='{.status.loadBalancer.ingress[0].ip}')
```

Comment:
For us to actually be testing this, we should have a separate step for "Test Workload," which includes the commented out section below.

Reply:
ok